cli/README.md

```diff
@@ -26,7 +26,7 @@ $ npm install -g @joystream/cli
 $ joystream-cli COMMAND
 running command...
 $ joystream-cli (-v|--version|version)
-@joystream/cli/0.8.0 linux-x64 node-v16.18.0
+@joystream/cli/0.10.0 darwin-x64 node-v14.18.0
```

Comment:
Could you merge-in the latest `ephesus` and re-generate the README in order to update the version as well?